src/analysis/vector/qgsgeometryanalyzer.cpp

```diff
@@ -1063,7 +1063,9 @@ QgsGeometry QgsGeometryAnalyzer::createOffsetGeometry( const QgsGeometry& geom,
   {
     if ( geom.type() == QgsWkbTypes::LineGeometry )
     {
-      GEOSGeometry* offsetGeom = GEOSOffsetCurve_r( geosctxt, ( *inputGeomIt ).asGeos(), -offset, 8 /*quadSegments*/, 0 /*joinStyle*/, 5.0 /*mitreLimit*/ );
+      GEOSGeometry* inputGeomItGeos = inputGeomIt->exportToGeos();
```

Comment:
Yeah however the method has more complicated logic that works directly with GEOS geometries, so I would need to rewrite it significantly (otherwise I would need to convert it back to GEOS again after the call to QgsGeometry::offsetCurve)